compatibility_lib/compatibility_lib/get_compatibility_data.py

```diff
@@ -81,15 +118,17 @@ def get_package_pairs(check_pypi=False, check_github=False):
     return self_packages, pair_packages
 
 
-def write_to_status_table(check_pypi=False, check_github=False):
+def write_to_status_table(
+        check_pypi=False, check_github=False, cloud_sql_proxy_path=None):
     """Get the compatibility status for PyPI versions."""
     # Write self compatibility status to BigQuery
     self_packages, pair_packages = get_package_pairs(check_pypi, check_github)
     results = checker.get_compatibility(
         packages=self_packages, pkg_sets=pair_packages)
     res_list = _result_dict_to_compatibility_result(results)
 
-    store.save_compatibility_statuses(res_list)
+    with connect_cloud_sql_proxy(cloud_sql_proxy_path):
```

Comment:
Could you just start this in the calling script?

Reply:
This is just the calling script we use.